tests/unit/tests_mvd3.cpp

```diff
@@ -18,11 +18,13 @@
  */
 #include <mvd/mvd3.hpp>
 #include <mvd/mvd_generic.hpp>
+
 #define BOOST_TEST_MAIN mvd3Parser
 #include <boost/test/included/unit_test.hpp>
```

Comment:
This looks dangerous. I've observed a lot of segfaults on the cluster due to mixing included unit test stuff and then linking against the lib. Wouldn't this be safer?
```suggestion
 #include <boost/test/unit_test.hpp>
```

Reply:
I agree with this suggestion. For the sake of getting this moving, could we attempt merging it?

Reply:
I can factor this out into a separate pr.